flexget/plugins/notifiers/discord.py

```diff
@@ -0,0 +1,138 @@
+                                'proxy_url': {'type': 'string', 'format': 'uri'},
+                            },
+                            'additionalProperties': False
+                        },
+                        'thumbnail': {
+                            'type': 'object',
+                            'properties': {
+                                'url': {'type': 'string', 'format': 'uri'},
+                                'proxy_url': {'type': 'string', 'format': 'uri'},
+                            },
+                            'additionalProperties': False
+                        },
+                        'provider': {
+                            'type': 'object',
+                            'properties': {
+                                'name': {'type': 'string'},
+                                'url': {'type': 'string', 'format': 'uri'},
+                            },
+                            'additionalProperties': False
+                        },
+                        'author': {
+                            'type': 'object',
+                            'properties': {
+                                'name': {'type': 'string'},
+                                'url': {'type': 'string', 'format': 'uri'},
+                                'icon_url': {'type': 'string', 'format': 'uri'},
+                                'proxy_icon_url': {'type': 'string', 'format': 'uri'},
+                            },
+                            'additionalProperties': False
+                        },
+                        'fields': {
+                            'type': 'array',
+                            'minItems': 1,
+                            'items': {
+                                'type': 'object',
+                                'properties': {
+                                    'name': {'type': 'string'},
+                                    'value': {'type': 'string'},
+                                    'inline': {'type': 'boolean'}
+                                },
+                                'required': ['name', 'value'],
+                                'additionalProperties': False
+                            }
+                        }
+                    },
+                    'additionalProperties': False
+                }
+            },
+        },
+        'required': ['web_hook_url'],
+        'additionalProperties': False
+    }
+
+    def notify(self, title, message, config):
+        """
+        Send discord notification
+
+        :param str message: message body
+        :param dict config: discord plugin config
+        """
+
+        web_hook = {
+            'content': message,
+            'username': config.get('username'),
+            'avatar_url': config.get('avatar_url'),
+            'embeds': config.get('embeds')
+        }
+
+        try:
+            r = requests.post(config['web_hook_url'], json=web_hook)
```

Comment:
variable `r` is not used for anything

```suggestion
            requests.post(config['web_hook_url'], json=web_hook)
```